plugins/org-react/src/plugin.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createComponentExtension,
+  createPlugin,
+} from '@backstage/core-plugin-api';
+
+import { rootRouteRef } from './routes';
+
+export const orgReactPlugin = createPlugin({
```

Comment:
This package isn't a plugin package and shouldn't export a plugin. It's a library, so you can export the component straight away without the extension wrapping.

Reply:
Updated!